sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param relationship_id: The Id of the relationship to get (if None all the relationship will be retrieved).
+        :type relationship_id: str
+        :return: An iterator like instance of either RelationshipCollection
+        :rtype: ~azure.core.paging.ItemPaged[~azure.digitaltwins.models.RelationshipCollection]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        return self._client.digital_twins.list_relationships(
+            digital_twin_id, 
+            relationship_name=relationship_id,
+            **kwargs
+        )
+
+    def list_incoming_relationships(self, digital_twin_id, **kwargs):
+        # type: (str, str, **Any) -> Iterable["models.IncomingRelationshipCollection"]
+        """Retrieve all incoming relationships for a digital twin.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no digital twin with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :return: An iterator like instance of either RelationshipCollection.
+        :rtype: ~azure.core.paging.ItemPaged[~azure.digitaltwins.models.IncomingRelationshipCollection]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        return self._client.digital_twins.list_incoming_relationships(
+            digital_twin_id, 
+            **kwargs
+        )
+
+    def publish_telemetry(self, digital_twin_id, payload, message_id=None, **kwargs):
+        # type: (str, object, str, **Any) -> None
+        """Publish telemetry from a digital twin, which is then consumed by
+           one or many destination endpoints (subscribers) defined under.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is no digital twin with the provided id.
+
+        :param digital_twin_id: The Id of the digital twin
+        :type digital_twin_id: str
+        :param payload: The telemetry payload to be sent
+        :type payload: object
+        :param message_id: The message Id
+        :type message_id: str
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        if not message_id:
+            message_id = uuid.UUID
+        timestamp = datetime.now
+        return self._client.digital_twins.send_telemetry(
+            digital_twin_id,
+            message_id,
+            telemetry=payload,
+            dt_timestamp=timestamp,
+            **kwargs
+        )
+
+    def publish_component_telemetry(self, digital_twin_id, component_path, payload, message_id=None, **kwargs):
```

Comment:
All of the operations that make API calls need to have the distributed trace decorator (and the async version on the async client).
This is as simple as importing it from `azure-core` and adding to the method signatures, as seen here:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

Reply:
Fixed